app/assets/javascripts/initializers/initializeTouchDevice.js

```diff
@@ -37,9 +41,7 @@ function initializeTouchDevice() {
   setTimeout(() => {
     removeShowingMenu();
     if (isTouchDevice) {
-      getById('navigation-butt').addEventListener('click', e =>
-        getClassList('navbar-menu-wrapper').toggle('showing'),
-      );
+      getById('navigation-butt').addEventListener('click', toggleMenu);
```

Comment:
I love this change, but, since I know how human minds work, in a six months time this could become again an anonymous function for any reason. Would you be willing to prepend this line with a comment explaining the reason why? Basically on the same notes of what you wrote in the issue description. Thank you very much!

Reply:
Done. Good call!